src/modules/commander/state_machine_helper.cpp

```diff
@@ -1010,6 +1010,13 @@ bool prearm_check(orb_advert_t *mavlink_log_pub, const vehicle_status_flags_s &s
 
 	}
 
+	if (!status_flags.condition_escs_valid) {
```

Comment:
@cmic0 This will fail for any vehicle which does not support the esc_status message.
Also check CI, SITL is actually complaining about ESC not being online :-D